build-info-extractor-npm/src/main/java/org/jfrog/build/extractor/npm/extractor/NpmBuildInfoExtractor.java

```diff
@@ -159,6 +162,10 @@ private void createTempNpmrc(Path workingDir, List<String> installationArgs) thr
         }
     }
 
+    private boolean isJsonOutputRequiered(List <String> installationArgs) {
+        return installationArgs.contains("--json") || installationArgs.contains("--json=true");
```

Comment:
1. `installationArgs.contains("--json=true")` infer `installationArgs.contains("--json")`. `installationArgs.contains("--json")` is enough.
2. This will not work on 
```
--json=false
--json false
--json        false
--json true
--json        true
```